eZ/Publish/Core/Repository/TrashService.php

```diff
@@ -310,17 +310,15 @@ public function findTrashItems(Query $query)
             throw new InvalidArgumentValue('query->limit', $query->limit, 'Query');
         }
 
-        $spiTrashItems = $this->persistenceHandler->trashHandler()->findTrashItems(
+        $spiTrashResult = $this->persistenceHandler->trashHandler()->findTrashItems(
             $query->filter,
             $query->offset !== null && $query->offset > 0 ? (int)$query->offset : 0,
             $query->limit !== null && $query->limit >= 1 ? (int)$query->limit : null,
```

Comment:
Until we don't have a method to get the number of trashed items in the API:  

```suggestion
            $query->limit !== null && $query->limit >= 0 ? (int)$query->limit : null,
```

Reply:
For the 0 vs null it would need a strict comparison right? or type check instead